src/core/qgsvectorlayertools.h

```diff
@@ -52,7 +52,7 @@ class CORE_EXPORT QgsVectorLayerTools : public QObject
      * \param feature         Updated feature after adding will be written back to this
      * \returns                True in case of success, False if the operation failed/was aborted
      *
-     * TODO QGIS 3: remove const qualifier
+     * TODO QGIS 4: remove const qualifier
```

Comment:
If you feel like cleaning up, none of these should be in the public docs or python docstrings

Reply:
nice, an easy win for more docs!